R/pkg/R/context.R

```diff
@@ -128,12 +135,15 @@ parallelize <- function(sc, coll, numSlices = 1) {
   objectSize <- object.size(coll)
 
   # For large objects we make sure the size of each slice is also smaller than sizeLimit
-  numSlices <- max(numSlices, ceiling(objectSize / sizeLimit))
-  if (numSlices > length(coll))
-    numSlices <- length(coll)
+  numSerializedSlices <- max(numSlices, ceiling(objectSize / sizeLimit))
+  if (numSerializedSlices > length(coll))
+    numSerializedSlices <- length(coll)
+
+  splits <- sort(rep(1: numSerializedSlices, each = 1, length.out = length(coll)))
```

Comment:
Can we add a small comment here on what the splits come out as (something like what you had in the PR comment)